pkg/daemon/gateway.go

```diff
@@ -434,12 +440,29 @@ func (c *Controller) setIptables() error {
 		if c.iptable[protocol] == nil {
 			continue
 		}
+		var matchset string
 		var iptableRules []util.IPTableRule
 		if protocol == kubeovnv1.ProtocolIPv4 {
 			iptableRules = v4Rules
+			matchset = "ovn40subnets"
 		} else {
 			iptableRules = v6Rules
+			matchset = "ovn60subnets"
 		}
+		for cidr, natip := range subnetNatips {
```

Comment:
Need to delete iptable rules if gw ip changes

Reply:
modified